packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -192,12 +186,14 @@ const projectContext = computed(() => props.project?.id.toString());
 const tabs = computed(() => tabStore.getTabs(projectContext.value) ?? []);
 const activeTabIndex = ref<number | null>(0);
 const openedAssetRoute = computed<Tab>(() => ({
-	assetName: props.assetName ?? '',
 	pageType: props.pageType,
-	assetId: props.assetId
+	assetId: props.assetId,
+	assetName: undefined // not used
```

Comment:
Can you elaborate ? I can't tell what perspective you are coming from and where you are going with this comment.

Reply:
Oh I misread .... I think no? this might be yet another quirky thing with the setup - that the Tab and URL are different but they are treated the same? As if we are using the URL to check the indexing of the existing Tabs, so the URL needs to be fudged to appear as if it is a Tab.

Reply:
hmmm...okay I think I actually can